lib/datasets-parquet/tlc/yellow-taxi-trip.rb

```diff
@@ -0,0 +1,112 @@
+require "parquet"
```

Comment:
Thank you!
In addition to this, I tried to add `datasets` to `lib/datasets-parquet.rb ` because of the same reason.
fix: [26f9672](https://github.com/red-data-tools/red-datasets-parquet/pull/3/commits/26f9672df63994b34b02ee8beca6f31d773f9bf9)